bpf/xdp.c

```diff
@@ -5,15 +5,15 @@
 
 // Test cases for bpf_xdp_adjust_head
 
-SEC("xdp/test_bpf_xdp_adjust_head_0") int test_bpf_xdp_adjust_head_0(struct xdp_md* ctx)
+SEC("xdp/test_bpf_xdp_adjust_head_0") int test_bpf_xdp_adjust_head_0(void** ctx)
```

Comment:
nit: should the section name prefix be `"xdp_test"` instead of `"xdp"`?

Reply:
Leaving this for when tests can be run with xdp for windows.